.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     parallelism: 3
     steps:
       - prometheus/setup_environment
-      - run: docker run --privileged linuxkit/binfmt:v0.8
+      - run: docker run --privileged linuxkit/binfmt:af88a591f9cc896a52ce596b9cf7ca26a061ef97
```

Comment:
Is there a tagged version that works?

Reply:
No, the binfmt image stopped doing tagged releases a few years ago.